src/adapter/src/coord/sequencer.rs

```diff
@@ -427,6 +431,7 @@ impl<S: Append + 'static> Coordinator<S> {
             depends_on,
             host_config,
         };
+        tracing::error!("I am in sequence_create_source");
```

Comment:
debug statement?

Reply:
It's amazing how I always triple-check before opening the PR, and one of them always goes through, sigh.

Reply:
i feel ya on that